style.css

```diff
@@ -0,0 +1,78 @@
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+header {
+  height: 100px;
+  margin: 10px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding: 10px;
+}
+
+.logo {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  font-weight: bolder;
+  color: #28352f;
+}
+
+.background-image {
+  background-image: url("asset/images/header-illsutration-mobile@2x.png");
+  min-height: 100vh;
+  background-size: cover;
+}
+
+.bar {
+  height: 10px;
+  width: 80px;
+  background-color: #36b37f;
+  margin: 7px;
+}
+
+.intro {
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: bolder;
+  font-size: 60px;
+  text-align: center;
+  padding: 0 30px 0 30px;
+  color: #172b4d;
+}
+
+.about-me {
+  margin: 150px 150px 0 150px;
```

Comment:
- [ ] You should use margins for mobile version in the figma design (0 24px) because it is affecting the layout on mobiles screens.
![mobile-ver-snapshot](https://user-images.githubusercontent.com/44057863/164618211-3d557076-00f5-4154-8a88-005b624afb4d.png)

Reply:
The correct margin has been used and the layout is better now. Thank you